python/cudf/cudf/core/series.py

```diff
@@ -301,7 +299,7 @@ def _constructor_expanddim(self):
     @classmethod
     @_cudf_nvtx_annotate
     def from_categorical(cls, categorical, codes=None):
-        """Creates from a pandas.Categorical
+        """Create from a pandas.Categorical.
```

Comment:
(Suggesting this, unless this is matching a pandas API docstring that I don't see)
```suggestion
        """Create a Series from a pandas.Categorical.
```